lib/std/Target.zig

```diff
@@ -1828,7 +1843,10 @@ pub const Cpu = struct {
         pub fn baseline(arch: Arch, os: Os) *const Model {
             return switch (arch) {
                 .amdgcn => &amdgcn.cpu.gfx906,
-                .arm, .armeb, .thumb, .thumbeb => &arm.cpu.baseline,
+                .arm, .armeb, .thumb, .thumbeb => switch (os.tag) {
+                    .@"3ds" => &arm.cpu.mpcore,
+                    else => &arm.cpu.baseline,
+                },
```

Comment:
```suggestion
                .arm => switch (os.tag) {
                    .@"3ds" => &arm.cpu.mpcore,
                    else => &arm.cpu.baseline,
                },
                .armeb, .thumb, .thumbeb => &arm.cpu.baseline,
```

Since `arm` seems to be the only arch tag used for the 3DS, we should only encode this knowledge for that tag. We have the same policy for the Apple stuff below (no `aarch64_be`), for example.